Cargo.toml

```diff
@@ -10,7 +10,7 @@ repository = "https://github.com/frewsxcv/rust-crates-index"
 license = "Apache-2.0"
 documentation = "https://docs.rs/crates-index/"
 edition = "2021"
-rust-version = "1.60"
+rust-version = "1.81"
```

Comment:
Can you help me understand this version bump?
I know `gix` shifted to 1.74 recently, but I could still compile this crate with this value being `1.60`.
So I think it's fair to undo this line-change.

Reply:
```suggestion
rust-version = "1.60"
```